File: estudos-alp-nelio-alves-main/c#/problema-crescente.cs
using System;
using System.Globalization;
using System.Security.Cryptography.X509Certificates;

namespace Crescente
{
    class Program
    {
        static void Main(string[] args)
        {
            CultureInfo CI = CultureInfo.InvariantCulture;

            int a, b;
            Console.Write("Digite dois números inteiros: ");
            string[] valores = Console.ReadLine().Split(' ');
            a = int.Parse(valores[0]);
            b = int.Parse(valores[1]);

            while (a != b)
            {
                if (a < b)
                {
                    Console.WriteLine("CRESCENTE!");
                }
                else
                {
                    Console.WriteLine("DECRESCENTE!");
                }
                Console.Write("Digite dois números inteiros: ");
                valores = Console.ReadLine().Split(' ');
                a = int.Parse(valores[0]);
                b = int.Parse(valores[1]);


                }
            }
        }
    }

File: estudos-alp-nelio-alves-main/cpp/problema-crescente.cpp
#include <iostream>

using namespace std;

int main()
{
    int x, y;
    cout << "Digite dois numeros inteiros: ";
    cin >> x >> y;

    while (x != y)
    {
        if (x < y)
        {
            cout << "CRESCENTE" << endl;
        }
        else
        {
            cout << "DECRESCENTE" << endl;
        }

        cout << "Digite outros dois numeros inteiros: ";
        cin >> x >> y;
    }

    return 0;
}